.github/workflows/android_envoy.yml

```diff
@@ -25,12 +25,13 @@ jobs:
       run: git submodule update --init --recursive
     - name: Clean and build
       env:
-        DNSTT_SERVER: ${{ secrets.DNSTT_SERVER }}
+        DNSTT_DOMAIN: ${{ secrets.DNSTT_SERVER }}
```

Comment:
why the mismatch between the keys?

Reply:
i was trying to avoid changing the secret in the repo, but it's not a big deal and i should probably do it